keyboards/sawnsprojects/satxri6key/keymaps/default/config.h

```diff
@@ -0,0 +1,8 @@
+#define RGB_MATRIX_KEYPRESSES // reacts to keypresses
```

Comment:
This should be moved into the keyboard's config.h file.

Otherwise, this file needs both a license header and an include guard (`#pragma once`)